### ISSUE TITLE
Help! Slowness. Then no access to LGPN-Ling for me. No log out possible. No access to the Editors.

### ISSUE THREAD

**Issue by Hippodamie** [eXistSolutions/LGPN-ling#525]
Dear Magdalena,

could you please help tomorrow morning? We will be at work from 9. am

It happened suddenly: it began to be slow when searching names and then was not possible anymore to edit.

Thanks in advance for your help.
Best,
Sophie

**Comment by tuurma** [comment]
Hi, I am unfortunately travelling with only phone for communication. Is the
server down?

śr., 23 sty 2019, 22:37: Hippodamie <notifications@github.com> napisał(a):

**Comment by Hippodamie** [comment]
It works well, now!  It went back yesterday night.
Have a good travel.
Could you please give very urgently the aswer to my email fo yesteray night (the ID Root form question)…?

Many thanks in advance.
Have a nice day,
best wishes,
Sophie

**Comment by Hippodamie** [comment]
Dear Magdalena,

I really need a quick answer to this question…
Could you please answer?

Many thanks,
with best wishes,
Sophie

Début du message réexpédié :

De: Sophie Minon <sophie.minon@ephe.sorbonne.fr<mailto:sophie.minon@ephe.sorbonne.fr>>
Objet: ID for Root: an urgent question you only can answer. Sorry to ask you in emergency! But I really need to know those very days where Violeta and I are at work together.
Date: 23 janvier 2019 21:30:37 UTC+1
À: Magdalena Turska <tuurma@gmail.com<mailto:tuurma@gmail.com>>

Dear Magdalena,

You did it!!! Great! Many very thanks.

Here is the question: it concerns in the subform the ID form (at left).

What is exactly the use of the ID? For what else would it be useful? Is it only an IT need, I mean for your own work? Or:

1. Will the future user of our Search interface need the ID to search for some Root, or not?

2. Will the ID be necessary for links with other projects?

I ask these questions because currently, as you can see, ID and Root forms do not exactly coïncide: very often, the last vowel of the plain Root is missing in the ID form.
It would be a hard work for us to make them coïncident, because, as you know, when we have 47 names with the same Root, if we add a vowel at the end of the ID form, we have to re-enter 47 times the new ID form in the Editors of the 47 different names. And sometimes we have, as for ευ more than 700 names...

To sum up:
if the ID forms (of course not Root forms themselves) remain as imperfect as currently, what kind of trouble can it cause: 1. For the building of the Search interface. 2. To build links with other projects?

Subsidiary question: is it a problem to have macrons in the ID forms like in αγορᾱ ?

Many thansk for your answers.
Good evening,
best wishes,
Sophie

**Comment by tuurma** [comment]
Dear Sophie,

assume we are talking about this form 

<img width="583" alt="image" src="https://user-images.githubusercontent.com/449468/51736122-92fa5b00-2089-11e9-9a48-fac7f3c939b2.png">

The ID is used only internally within the system, it is not visible in any other place. Thus, it doesn't matter what it is. It could be even a number, but I thought it is just more human readable if we leave it as close to the form as possible.

For the search interface it doesn't matter - what is searched is the root form, not the id.
Linking with other projects, they should use the id but again, it doesn't matter how the id itself looks like, only to which form it links.

Macrons are not a problem. We cannot have spaces, parentheses etc.

**Comment by Hippodamie** [comment]
Dear Magdalena,
exactly.

**Comment by Hippodamie** [comment]
Μany, many thanks for your answer (I did not see at forst it was written under the screenshot!)
Best,
Sophie
